FireRender.Maya.Src/FireRenderObjects.cpp

```diff
@@ -1520,7 +1520,12 @@ void FireRenderMesh::GetShapes(std::vector<frw::Shape>& outShapes)
 	{
 		bool deformationMotionBlurEnabled = IsMotionBlurEnabled(MFnDagNode(dagPath.node())) && TahoeContext::IsGivenContextRPR2(context) && !context->isInteractive();
 		unsigned int motionSamplesCount = deformationMotionBlurEnabled ? context->motionSamples() : 0;
+
+		//disable callbacks to set objects dirty
+		context->disableSetDirtyObjects(true);
```

Comment:
Maybe this should be done inside deformation processing? (inside mesh translator)
What do you think?